internal/crossdock/client/oneway/raw.go

```diff
@@ -30,14 +30,19 @@ import (
 )
 
 // Raw starts an http run using raw encoding
-func Raw(t crossdock.T, dispatcher yarpc.Dispatcher) {
+func Raw(t crossdock.T, dispatcher yarpc.Dispatcher, serverCalledBack <-chan []byte, callBackAddr string) {
+	assert := crossdock.Assert(t)
 	fatals := crossdock.Fatals(t)
 
-	client := raw.New(dispatcher.ClientConfig("oneway-test"))
-	ctx := context.Background()
-
+	client := raw.New(dispatcher.ClientConfig("oneway-server"))
 	token := []byte(getRandomID())
-	ack, err := client.CallOneway(ctx, yarpc.NewReqMeta().Procedure("echo/raw"), token)
+
+	ack, err := client.CallOneway(
+		context.Background(),
+		yarpc.NewReqMeta().
+			Procedure("echo/raw").
```

Comment:
It is, but this oneway client/server interaction is essentially an 'echo' service. Though this echoing involves two RPCs, the data you give it is the same you get back so I felt it was still fitting to name it echo